src/include/ceph_features.h

```diff
@@ -35,6 +35,7 @@
 #define CEPH_FEATURE_OSD_HBMSGS     (1ULL<<28)
 #define CEPH_FEATURE_MDSENC         (1ULL<<29)
 #define CEPH_FEATURE_OSDHASHPSPOOL  (1ULL<<30)
+#define CEPH_FEATURE_NEW_ENCODING   (1ULL<<30)
```

Comment:
Heh, I know I made a reference to maybe letting it share with another feature, but I meant if we had another feature go in during the same dev release, not trying to match it up with an existing one! Just give it an unused number so we can disambiguate between brand new clients that support putting the pgid at the front, versus those that understand the new OSDMap encoding that this covers. ;)